definition_of_done.md

```diff
@@ -0,0 +1,11 @@
+# LeApp - Definiton of done
+
+ * [ ] The acceptance criteria described in the Task is met
+ * [ ] The code is approved and merged into the repository
+ * [ ] The documentation is updated
+ * [ ] Unit tests that cover the new code are written and passing
+ * [ ] Integration tests are passing
+ * [ ] The ticket contains link(s) with the results of the Task (e.g., PR, document, etc.).
+
+**If the work needs to be documented, discussed or confirmed:**
+ * [ ] An e-mail was sent to leapp-devel list 
```

Comment:
please let's remove this, if task needs to be discussed it's not done, so this shouldn't be in DoD because it describes different process.

Reply:
+1